.palantir/revapi.yml

```diff
@@ -16,9 +20,15 @@ acceptedBreaks:
     - code: "java.method.addedToInterface"
       new: "method java.util.List<org.apache.iceberg.StatisticsFile> org.apache.iceberg.Table::statisticsFiles()"
       justification: "new API method"
+    - code: "java.method.addedToInterface"
+      new: "method org.apache.iceberg.TableScan org.apache.iceberg.TableScan::useRef(java.lang.String)"
+      justification: "Adding table scan APIs to support scanning from refs"
     - code: "java.method.removed"
       old: "method java.lang.Iterable<org.apache.iceberg.DataFile> org.apache.iceberg.Snapshot::addedFiles()"
       justification: "Deprecations for 1.0 release"
+    - code: "java.method.removed"
+      old: "method java.lang.Iterable<org.apache.iceberg.DataFile> org.apache.iceberg.Snapshot::deletedFiles()"
+      justification: "Deprecations for 1.0 release"
```

Comment:
Can you revert the changes that moved these around?

Reply:
These were moved around due to the API change in `NestedFields`. It should go away once we fix the access modifiers for `optional` and the constructor. Access modifiers were changed to accommodate keeping the new APIs private, but we found there was no good solution for that, so we will just go with making them new API public (as it is the case in the end state). This will take care of keeping this file intact as well.

Reply:
Right now this only contains
```
    - code: "java.class.defaultSerializationChanged"
      old: "class org.apache.iceberg.types.Types.NestedField"
      new: "class org.apache.iceberg.types.Types.NestedField"
      justification: "Serialization across versions is not supported"
```
I think this should be expected as we added a new API.

Reply:
Yes, that sounds fine.